### ISSUE TITLE
dark mode pages

### ISSUE THREAD

**Issue by Rrhyast** [minbrowser/min#2000]
Enable dark mode for web page because my eyes are bleeding. Please!

**Comment by atxr** [comment]
Yes it would be very nice to have something like https://darkreader.org/ in the min browser

**Comment by Parmjot-Singh** [comment]
https://github.com/PalmerAL/min-dark-mode-userscript
Please mark this as closed.

**Comment by FuryFiber** [comment]
https://gist.github.com/kfur/266c456dd69072eb7533f457ee5f18a0 this script does the same thing but just slightly better in my opinion. (it says its for safari but it seems to work perfectly fine with min)

> https://github.com/PalmerAL/min-dark-mode-userscript
this just inverts color on every website right?

**Comment by Trit34** [comment]
Up! I just discovered Min since yesterday and I wish it has a global dark mode for webpages too, as Chromium browsers can have with their “#enable-force-dark” flag (but set to “Enabled with selective inversion of non-image elements”). Since Min is built upon Electron (and Electron is Chromium, basically), it could be easy to add an option to enable and disable it at will in the Preferences, I guess (it does not matter if I have to relauch the browser). Even Vivaldi on Android can do that.
